app-emulation/firecracker-bin/firecracker-bin-1.5.1.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 2021-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit linux-info
+
+DESCRIPTION="Secure and fast microVMs for serverless computing (static build)"
+HOMEPAGE="https://firecracker-microvm.github.io https://github.com/firecracker-microvm/firecracker"
+SRC_URI="
+	amd64? (
+		https://github.com/firecracker-microvm/firecracker/releases/download/v${PV}/firecracker-v${PV}-x86_64.tgz
+	)
+	arm64? (
+		https://github.com/firecracker-microvm/firecracker/releases/download/v${PV}/firecracker-v${PV}-aarch64.tgz
+	)"
+
+LICENSE="|| ( Apache-2.0 MIT Apache-2.0-with-LLVM-exceptions ) MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+
+RESTRICT="test strip"
+
+RDEPEND="acct-group/kvm"
+
+QA_PREBUILT="usr/bin/firecracker
+	usr/bin/jailer
+	usr/bin/rebase-snap
+	usr/bin/seccompiler-bin"
```

Comment:
```
 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/bin/cpu-template-helper
 * /usr/bin/snapshot-editor
 * QA Notice: Files built without respecting LDFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/bin/cpu-template-helper
 * /usr/bin/snapshot-editor
```

Reply:
Thank you, I missed that.
Since `pkgcheck scan ...` and `ebuild ... install` don't report this: What's the best approach to get notified of things like this?

Reply:
These come from CFLAG and LDFLAG checks, 
https://wiki.gentoo.org/wiki/Package_testing#make.conf_and_test.conf_files